src/main/java/ch/uzh/ifi/hase/soprafs22/controller/UserController.java

```diff
@@ -138,21 +138,19 @@ public void updateUser(
     userService.updateUser(userInput); // this throws errors 404 and 409
   }
 
-  @PostMapping("/users/usernames")
+  @GetMapping("/users/usernames")
   @ResponseBody
```

Comment:
this leads to an error. 1) client site is still post 2) also get en error when i change the client to get

Reply:
Call it with GET users/usernames?available=yourtestusername.